### PR TITLE
Maven updates

### PR DESCRIPTION
**Pull request by rleigh-codelibre**
This PR cherry-picks the `east` maven update commits for use on `develop`.

Testing: check jobs remain green.

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -5,6 +5,11 @@ environment:
 
 # Note that only Oracle JDK is provided.
   matrix:
+    - java: 9
```
Comment (sbesson) [inline .appveyor.yml]: Main concerns about adding the Appveyor  matrix is: are we going to reach the limits of our usage and start to block PR checks once merged into`develop`? Do we have metrics about usage, wait & completion times?

Comment (rleigh-codelibre) [reply .appveyor.yml]: I have dropped two ant builds from both travis and appveyor so that it only builds with the primary supported jdk (8).

**Comment by joshmoore** [comment]
I'm unclear what the goal of opening this against develop at the moment is. My understanding from our discussion yesterday was that we were going to focus on getting `east` into a state that it was ready for merging into `develop`. Closing for the moment.

**Comment by rleigh-codelibre** [comment]
This *is* getting it ready for merging, by collecting together the maven parts of east and getting them onto develop.  This reduces `east` to the subresolution-specific changes only so that it's easy to review the delta between them.  That is to say, it is taking the non-subresolution-related changes out of the picture.

**Comment by joshmoore** [comment]
Appreciate the effort, but as we discussed, I don't think this is how we see the reintegration working. If we decide to go this route, we can always reopen. For the moment, let's assume that the review will take place via a single larger merge, rather than a rebase strategy.

**Pull request closed without merging**